web-example/README.md

```diff
@@ -5,4 +5,10 @@
 1. Ensure you have the [https://github.com/cobrowseio/cobrowse-sdk-js-examples](https://github.com/cobrowseio/cobrowse-sdk-js-examples) repo cloned locally.
 2. `cd web-example`
 3. Replace `trial` seen at line 44 of [./src/hooks/useCobrowse.js`](https://github.com/cobrowseio/cobrowse-sdk-js-examples/blob/1a35aef3428a01fcad4b785e40a7454b421e50dd/web-example/src/hooks/useCobrowse.js#L44) to your own licnese key. (Register a new account at [https://cobrowse.io/register](https://cobrowse.io/register)).
-4. Run `npm start` from within the `web-example` directory
\ No newline at end of file
+4. Run `npm start` from within the `web-example` directory
+
+## Query Parameters
+
+The web demo can be used with api, license, registration and custom_data parameters. Different integrations are also supported out of the box: salesforce, zendesk, genesys, freshdesk, talkdesk.
```

Comment:
Do we want to add a note here suggesting this should not be done in production applications?

https://github.com/cobrowseio/cobrowse-sdk-js-examples/blob/977bb75a00d786f228a53a180e73d944efa9d9e1/connection-last-alive-check.html#L47